### PR TITLE
[feat][refactor] variables

### LINKED ISSUE #16: [Refactor][Replace variables with hard-coded units]
- Hardcoded units are used in components and documentation for font size, font-weight, and border-radius
- Uneven numerical values are used(i.e not in order of 4)

### PR DESCRIPTION
**Pull request by abhij1607**
- Declared variables for font size, font-weight
- replaced the hardcoded units with variables

### CODE REVIEW

File: docs/css/base.css [hunk 1]
```diff
@@ -22,4 +22,23 @@
   --danger-light-color: #f8d7da;
   --warning-light-color: #fff385;
   --info-light-color: #cff4fc;
+
+  --size-2xs: 4px;
```
Comment (GuptaEesh) [inline docs/css/base.css:26]: 2xs? I didn't understand it as xs=0.8rem and 2xs=4px!!??

**Review by GuptaEesh — COMMENTED** [review]
Apart from the anomaly I pointed out everything looks great.

File: docs/css/base.css (hunk 1 above)
Comment (abhij1607) [reply docs/css/base.css:26]: xs means xtra small and 2xs means xxs(extra extra small)

Comment (GuptaEesh) [reply docs/css/base.css:26]: OHHH!! What a naming!

**Review by GuptaEesh — APPROVED** [review]

**Review by ZuberDunge — APPROVED** [review]
Good naming of variables. Good to merge from my side.

**Comment by abhij1607** [comment]
changed hardcoded values to variables, this fixes #16

**Pull request merged**